### ISSUE TITLE
A question about code

### ISSUE THREAD

**Issue by peter943** [Ontheway361/dul-pytorch#3]
Wonderful job！I studied your paper these days, which is very enlightening to me. I try to use the data uncertainty in my research.
   I have a question about the caculation of the variance. In your code, you use the 
var = F.normalize(torch.exp(self.var_head(x))), to process the variance predicted by network. I guess the  torch.exp() is used to guarantee the positive of the variance. The F.normalize() I guess is simliar to the L2-normalization for the feature to make the network converge better.
   I do not know this understanding is right?  Thank you.

**Comment by Ontheway361** [comment]
yes, you're right!

**Comment by Ontheway361** [comment]
@peter943 yes, you're right!

**Comment by peter943** [comment]
@Ontheway361 Thank you for your kind help. I was wondering whether it is possible that I could ask you a more question. I am working for recognition but not face recognition. The network I use maybe different from yours. 
      For face recognition, adding data uncertainty module to the backborne would improve the performance from your paper. I wonder whether the network difference would significantly influence this performance gain. Thank you !
